pkg/storage/action.go

```diff
@@ -49,3 +50,28 @@ func (s *Store) CreateAction(ctx context.Context, action *models.Action) (*model
 	}
 	return action, nil
 }
+
+// GetActionsByRequestID fetches actions for a particular request
+func (s *Store) GetActionsByRequestID(ctx context.Context, id uuid.UUID) ([]models.Action, error) {
+	actions := []models.Action{}
+	const fetchActionsByRequestIDSQL = `
+		SELECT
+		       *
+		FROM
+		     actions
+		WHERE actions.intake_id=$1
+	`
+	err := s.db.Select(&actions, fetchActionsByRequestIDSQL, id)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(
+			fmt.Sprintf("Failed to fetch actions %s", err),
```

Comment:
Why build a string with the error, rather than just pass the error as it's own field? E.g.:
```go
loggger.Error("failed to fetch actions", zap.Error(err), zap.String("intakeID", id.String())
```